ibm/service/vpc/resource_ibm_is_instance.go

```diff
@@ -4253,3 +4259,19 @@ func GetInstanceMetadataServiceOptions(d *schema.ResourceData) (metadataService
 	}
 	return nil
 }
+
+// This function takes two lists and returns the difference between the two lists
+// listdifference([1,2] [2,3]) = [1]
```

Comment:
We already have this function https://github.com/IBM-Cloud/terraform-provider-ibm/blob/f7043dd036eefba25a2d8ca27dae0625910a0347/ibm/service/classicinfrastructure/resource_ibm_multi_vlan_firewall.go#L415
here also can we move to structure.go and use for both

Reply:
yes just move to common place and update that also

Reply:
Changes incorporated

Reply:
<img width="536" alt="Screenshot 2023-04-20 at 7 00 53 AM" src="https://user-images.githubusercontent.com/127872893/233234704-4a0a2494-bd75-4c2d-8343-4e9ebe114991.png">